api/mesh/v1alpha1/zone_ingress.proto

```diff
@@ -23,6 +23,7 @@ message ZoneIngress {
   option (kuma.mesh.resource).scope_namespace = true;
   option (kuma.mesh.resource).has_insights = true;
   option (kuma.mesh.resource).short_name = "zi";
+  option (kuma.mesh.resource).proxy_type = "ingress";
```

Comment:
I'm still not sure if it's a good idea to stick to old values like `ingress` and `egress`. Now they're user facing and it might be difficult for users to associate `ingress` with `ZoneIngress` type.

Should we go with `ZoneIngress` and `ZoneEgress` values?

Reply:
If we want to use form `ZoneIngress` instead of `ingress` then maybe we can just don't add this proxy type field and just use `ResourceTypeDescriptor.name`. Also for a dataplane should we use `Sidecar`as we now use in policy `proxyType` field. 

Also I am not sure if can that easily change values of `ProxyType` we use atm, should we then have new type for new ProxyType values?